src/Bigcommerce/Api/Client.php

```diff
@@ -413,6 +418,32 @@ public static function getAuthToken($object)
         return $connection->post(self::$login_url . '/oauth2/token', $context);
     }
 
+    public static function getCustomerLoginToken($id, $redirectUrl = '', $requestIp = '')
+    {
+        if (empty(self::$client_secret)) {
+            throw new Exception('Cannot sign customer login tokens without a client secret');
+        }
+
+        $payload = array(
+            'iss' => self::$client_id,
+            'iat' => time(),
+            'jti' => bin2hex(random_bytes(32)),
+            'operation' => 'customer_login',
+            'store_hash' => self::$store_hash,
+            'customer_id' => $id
+        );
+
+        if (!empty($redirectUrl)) {
```

Comment:
[JWT already specifies an order](https://tools.ietf.org/html/rfc7519#section-11.2) for signing the payload.